### ISSUE TITLE
Integrate web-native rendering-library as a web-worker which provides a simple internal API

### ISSUE THREAD

**Issue by LisaPich** [SYP-AHIF-2023-24-25/SolarDoc#46]
Integrate the web-native render library for rendering Asciidoc (#51) by using a web-worker to run alongside the frontend main thread.

Depends on #45

**Comment by Luna-Klatzer** [comment]
Depends on #51 and therefore is postponed until further notice.

**Comment by Luna-Klatzer** [comment]
Closed as of release `v0.3.0`, as browserify has been officially _deprecated_ internally and is going to be removed as of `v0.4.0`.

As such this task will be closed as implementation on it will not be resumed.
